docs/v0.6.x/s3-walkthrough.md

```diff
@@ -129,6 +129,17 @@ The file `index.html` is not there because ember-cli-deploy-s3 does not upload i
 
 The setting `filePattern` tells ember-cli-deploy-s3 what files should be uploaded. I have just taken the default value and added `html` to the list of extensions. Try deploying again and you'll see that `index.html` is deployed this time.
 
+Next, make sure your router is using `hash` for its `locationType`.  This is important so that your files can be found correctly.
+
+Your router will likely pull this configuration from your `config/environment.js` file.
+
+    var ENV = {
+      // ...
+      locationType: 'hash'
+    }
+    
+Deploy again if your `locationType` was previously anything besides `hash` (including `auto`).
```

Comment:
That would put it before we introduce `index.html` and the problem that the note is meant to solve.  Not a great solution, but maybe it's better than an extra sentence.